src/rubrix/server/apis/v0/handlers/text2text.py

```diff
@@ -130,6 +130,7 @@ def bulk_records(
 )
 def search_records(
     name: str,
+    id_from: Optional[str] = None,
```

Comment:
Since it doesn't affect the loading of datasets with `rb.load`, I prefer to leave this parameter out of the search for now. 

The motivation is that we should solve the problem when searching with a sort provided to the API. With the proposed implementation, provided sorting has no effect and that worries me. 

It is an issue that can be resolved, but it is not easy. That's why I prefer to approach it independently.